packages/astro/src/core/messages.ts

```diff
@@ -104,6 +104,13 @@ export function serverShortcuts({ key, label }: { key: string; label: string }):
 	return [dim('  Press'), key, dim('to'), label].join(' ');
 }
 
+export function newVersionAvailable({ latestVersion }: { latestVersion: string }) {
+	const badge = bgYellow(black(` update `));
+	const headline = yellow(`▶ New version of Astro available: ${latestVersion}`);
+	const details = `  Run ${cyan('npx @astrojs/upgrade')} to update`;
```

Comment:
Is it possible to update the command based on the package manager? eg `pnpm dlx @astrojs/upgrade` for pnpm

Reply:
It is possible yes!

Reply:
Agree that this would be nice to handle!